blueprints/dumb/files/__test__/components/__name__.spec.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react'
+import <%= pascalEntityName %> from 'components/<%= pascalEntityName %>/<%= pascalEntityName %>'
```

Comment:
I changed it to nest dumb components in an extra folder, so it is currently correct. I have no updated views yet, though.